src/posthog-core.ts

```diff
@@ -1977,12 +1980,15 @@ export class PostHog {
         this.consent.optInOut(true)
         this._sync_opt_out_with_persistence()
 
-        if (!isUndefined(options?.captureEventName) && !options?.captureEventName) {
-            // Don't capture if captureEventName is null or false
-            return
+        // Don't capture if captureEventName is null or false
+        if (isUndefined(options?.captureEventName) || options?.captureEventName) {
+            this.capture(options?.captureEventName ?? '$opt_in', options?.captureProperties, { send_instantly: true })
         }
 
-        this.capture(options?.captureEventName ?? '$opt_in', options?.captureProperties, { send_instantly: true })
+        if (this.config.capture_pageview && !this._initialPageviewCaptured && document) {
+            this._initialPageviewCaptured = true
+            this.capture('$pageview', { title: document.title }, { send_instantly: true })
+        }
```

Comment:
Could we move this block into a private method like `captureInitialPageview` instead. that way it isn't likely to get forgotten about later in one place or the other

Reply:
Thanks @benjackwhite! I added a private method.